SQL/New_patches/2018-09-07_EEG_tables.sql

```diff
@@ -0,0 +1,265 @@
+-- Create a physiological_modality table
+CREATE TABLE `physiological_modality` (
+  `PhysiologicalModalityID` INT(5) UNSIGNED NOT NULL AUTO_INCREMENT,
+  `PhysiologicalModality`   VARCHAR(50)         NOT NULL UNIQUE,
+  PRIMARY KEY (`PhysiologicalModalityID`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
```

Comment:
in mysql "utf8" isn't. These needs to be "utf8mb4" in order to be utf8. ..

Reply:
@driusan Thanks! Done.

Reply:
seems to still be utf8, not utf8mb4? Did you forget to push?

Reply:
yeah, we have been experiencing with Mélanie and Tom regarding this. I did change everything to `utf8mb4` but since `physiological_file` is referencing to the `session` and `ImagingFileTypes` tables that are `utf8`, the foreign keys cannot be created and it fails creating that table and all tables that references it.
We also tried specifying the charset in the attribute and it failed too. We did not find any solution that seemed to work unless putting all table to utf8 like the rest of the schema.
We tried but in the end we had to revert back to utf8... :(

Reply:
I think the right change is to fix the session and ImagingFileTypes tables, then, but okay, we'll leave it for now.